### DISCUSSION TITLE
Multiple conditions in template logic

### DISCUSSION THREAD

**Discussion by Schweber** [JanDeDobbeleer/oh-my-posh#5290] [Troubleshoot] [answered]
I would like the battery segment to only show if there are no errors **and** the battery is not charging. However, i'm struggling with the logic system of `oh-my-posh` and have only found a way to implement one condition because i don't understand if and how i can employ the `and` keyword together with other keywords for `if`.

The following works but i don't know how i can add `not .Error` to it so that both conditions are checked:

`template = " {{ if ne \"Charging\" .State.String }} {{ .Percentage }}{{ end }} "`

How can i combine both conditions into one clause?

**Comment by JanDeDobbeleer** [comment, marked as answer]
```toml
template = "{{ if and (ne \"Charging\" .State.String) (not .Error) }} {{ .Percentage }} {{ end }}"
```

**Reply by Schweber** [reply]
Thank you!
